users/serializers/company_bind_serializer.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import serializers
+from django.core.exceptions import ValidationError
+from validation.validate_names import validate_forbidden_names
+from validation.validate_string_fields import validate_max_length
+
+
+class CompanyBindingSerializer(serializers.Serializer):
+    """ Serializer for validating company binding requests. """
+    company_name = serializers.CharField(max_length=254)
+    company_type = serializers.ChoiceField(choices=['startup', 'investor'])
+
+    def validate_company_name(self, value):
+        """
+        Validate company name using existing validation functions.
+        """
+        value = value.strip()
```

Comment:
The input value is not normalized to a string, which can cause exceptions if an integer is provided. Convert value to string before stripping.
```
value = str(value).strip()
```